convlab2/nlg/template/multiwoz/nlg.py

```diff
@@ -202,29 +202,29 @@ def _manual_generate(self, dialog_acts, template):
                 for slot, value in slot_value_pairs:
                     if value in ["do nt care", "do n't care", "dontcare"]:
                         sentence = 'I don\'t care about the {} of the {}'.format(slot.lower(), dialog_act.split('-')[0].lower())
-                    elif self.is_user and dialog_act.split('-')[1] == 'Inform' and slot == 'Choice' and value == 'any':
+                    elif self.is_user and dialog_act.split('-')[1] == 'inform' and slot == 'choice' and value == 'any':
```

Comment:
Is value.lower() == 'any' more robust? Or, probably we can make this code snippet segregated from the cases in the input, by adding  slot_lower, value_lower, dialog_act_lower = slot.lower(), value.lower(), dialog_act.lower() after line 202, with corresponding changes on other lines associated with string comparison?

Reply:
Good point. slot and dialog_act are already lower cased in 117 and 119 of current PR. Changed the implementation to use value_lower instead of calling .lower() on each comparison.